From: KevinDuan <duankaiwen@kylinos.cn>
Date: Wed, 2 Oct 2024 02:53:58 +0000
Subject: Translated using Weblate (Kyrgyz)

Currently translated at 100.0% (5 of 5 strings)

Translation: openkylin-nile-new/kylin-os-manager-tool-box
Translate-URL: http://weblate.openkylin.top/projects/openkylin-nile-new/kylin-os-manager-tool-box/ky/
---
 plugins/tool-box/translations/kylin-os-manager-tool-box_kk.ts | 6 +++---
 plugins/tool-box/translations/kylin-os-manager-tool-box_ky.ts | 6 +++---
 2 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/plugins/tool-box/translations/kylin-os-manager-tool-box_kk.ts b/plugins/tool-box/translations/kylin-os-manager-tool-box_kk.ts
index 6d5d28b..de52563 100644
--- a/plugins/tool-box/translations/kylin-os-manager-tool-box_kk.ts
+++ b/plugins/tool-box/translations/kylin-os-manager-tool-box_kk.ts
@@ -6,12 +6,12 @@
     <message>
         <location filename="../appitem.cpp" line="114"/>
         <source>coming soon</source>
-        <translation>Реттелген күйде қалу</translation>
+        <translation>ساقتاپ تۇر</translation>
     </message>
     <message>
         <location filename="../appitem.cpp" line="115"/>
         <source>More tools are coming soon</source>
-        <translation>Жақын арада қосымша жүйелік виджеттер келе жатыр</translation>
+        <translation>الٸدە كوپ سەستيما كشكەنە قۇرالى جاقىندا اراسىندا جارناما قىلىندى</translation>
     </message>
 </context>
 <context>
@@ -19,7 +19,7 @@
     <message>
         <location filename="../plugininterface.cpp" line="35"/>
         <source>ToolBox</source>
-        <translation>Қазына кеудесі</translation>
+        <translation>قۇرال ساندىعى</translation>
     </message>
 </context>
 <context>
diff --git a/plugins/tool-box/translations/kylin-os-manager-tool-box_ky.ts b/plugins/tool-box/translations/kylin-os-manager-tool-box_ky.ts
index 8af2395..3d6abd7 100644
--- a/plugins/tool-box/translations/kylin-os-manager-tool-box_ky.ts
+++ b/plugins/tool-box/translations/kylin-os-manager-tool-box_ky.ts
@@ -6,12 +6,12 @@
     <message>
         <location filename="../appitem.cpp" line="114"/>
         <source>coming soon</source>
-        <translation>Кунт коюп уккула</translation>
+        <translation>ساقتاپ  تۇرۇڭ</translation>
     </message>
     <message>
         <location filename="../appitem.cpp" line="115"/>
         <source>More tools are coming soon</source>
-        <translation>Дагы системалык виджеттер жакында келе жатат</translation>
+        <translation>داعى ەلە كۅپ ساامالىق  كىچىك  قۇرالى جاقىن ارادا جارنااما  جاسالات</translation>
     </message>
 </context>
 <context>
@@ -19,7 +19,7 @@
     <message>
         <location filename="../plugininterface.cpp" line="35"/>
         <source>ToolBox</source>
-        <translation>Кенч көкүрөгү</translation>
+        <translation>قۇرال  ساندىعى</translation>
     </message>
 </context>
 <context>
